### DISCUSSION TITLE
non-passive event listeners v6

### DISCUSSION THREAD

**Discussion by brianp70x** [fabricjs/fabric.js#8062] [General]
can anything be done about this chrome warning?

[Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

fabric.util.addListener | @ | fabric.min.js?ver=6.0.0:1

  | addOrRemove | @ | fabric.min.js?ver=6.0.0:1
  | _initEventListeners | @ | fabric.min.js?ver=6.0.0:1
  | _initInteractive | @ | fabric.min.js?ver=6.0.0:1
  | initialize | @ | fabric.min.js?ver=6.0.0:1
  | i | @ | fabric.min.js?ver=6.0.0:1

thanks

**Comment by ShaMan123** [comment]
I wouldn't bother.
I think it should be non passive because it takes over scrolling.

**Reply by ShaMan123** [reply]
It's chrome trying to help you with perf. But irrelevant here I believe
